sunpy/map/sources/soho.py

```diff
@@ -279,6 +279,13 @@ def spatial_units(self):
 
         return SpatialPair(u.Unit(cunit1), u.Unit(cunit2))
 
+    @property
+    def unit(self):
```

Comment:
This whole property seems like it's not doing anything - is there a reason you can't remove it?

Reply:
I want to avoid it calling the super version as it will error from the fits validation unit check.

Reply:
What's the error you're getting? And if it's erroring now, won't it also error when the unit is added to astropy, so we'll have to fix the `._parse_fits_unit` method at some point any way?

Reply:
The error is that Maxwell is not a valid FITS unit. So to avoid it being called, I override the unit on the map source.

My impression and Stuart (@cadair) will correct me that the code in mapbase is only going to allow FITS compliance and anything else has to go into the map source.

Reply:
Will do.